app/src/main/kotlin/maru/app/MaruApp.kt

```diff
@@ -215,12 +216,18 @@ class MaruApp(
       }
     val forkTransitionSubscriptionManager = InOrderFanoutSubscriptionManager<ForkSpec>()
     forkTransitionSubscriptionManager.addSyncSubscriber(p2pNetwork::handleForkTransition)
+    val elDelegatedConsensusFactory =
+      ElDelegatedConsensusFactory(
+        ethereumJsonRpcClient = validatorELNodeEthJsonRpcClient.eth1Web3j,
```

Comment:
not related to logic: but why here we called it `validatorELNodeEthJsonRpcClient.eth1Web3j` where the El node could be from follower

Reply:
We need a node on which we have the most control and which provides the strongest consistency with Maru's beacon chain. That's the payload validator el node. Also, that's the only mandatory node. Followers list may be empty